customize-robots.Rmd

```diff
@@ -90,53 +116,84 @@ These previews do NOT incorporate any changes influenced by any changes to the D
 
-To resolve those spelling errors, click on the link with the errors on the autogenerated comment.
 Some of these errors may be things that the spell check doesn't recognize for example: `ITCR` or `DaSL`.
 If it's a 'word' the spell check should recognize, you'll need to add this to the dictionary.
 
 Go to the `resources/dictionary.txt` file. Open the file and add the new 'word' to its appropriate place (the words are in alphabetical order).
 After committing the changes to `resources/dictionary.txt` on your branch, the spell check status check should be successfully passed.
 
+The PR comment included below shows an example of a check that passed (spell check), and a check that failed (the URL check) with the option to download detailed error report(s).
+
+![](https://raw.githubusercontent.com/ottrproject/ottrproject.github.io/kweav-actionsDocsUpdate/resources/screenshots/check_results.png)
+
+When all checks pass, the option to download error reports is no longer included, instead displaying a message that the URL and spell checks passed.
+
+![](https://raw.githubusercontent.com/ottrproject/ottrproject.github.io/kweav-actionsDocsUpdate/resources/screenshots/check_results_pass.png)
+
 ### Style code
 
 In the `config_automation.yml` file it is set by:
 ```
-style-code: yes
+style-code: true
 ```
 The `styler` package will style R code in all Rmds. Style changes will automatically be committed back to your branch.
 
 ### Docker testing
 
 In the config_automation.yml file it looks like:
 ```
-docker-test: no
+docker-test: false
 ```
-By default it is set to `no` which means it won't run automatically unless you change this to `yes`.
+By default it is set to `false` which means it won't run automatically unless you change this to `true`.
 This is only relevant if you have [your own Docker image you are managing for your course](https://www.ottrproject.org/customize-docker.html). If changes are made to Docker-relevant files: `Dockerfile`, `install_github.R`, or `github_package_list.tsv`, this will test re-build the Docker image. If it is successfully built, then it makes sense to merge it to `main` but the Docker Image will not be pushed to `Dockerhub` automatically. [Follow these instructions to push your Docker image to Dockerhub](https://www.ottrproject.org/customize-docker.html#Testing_a_modified_Docker_image).
 
 <br>
 
 ## Rendering actions
 
 Upon merging changes to any `Rmd` or `assets/` folder to `main`, the course material will be automatically re-rendered.
-By default, all rendering steps will be run. But depending on the needs of your course, you can turn these on and off by going to the `config_automation.yml` file and switching options to `yes` or `no`.
+
+For rendering a website version of the course (or your website), rather than `true` or `false`, the options are `rmd`, `rmd_web`, `quarto` or `quarto_web`.
```

Comment:
```suggestion
For rendering a website version of the course (or your website), rather than `true` or `false`, the options are `rmd`, `rmd_web`, `quarto` or `quarto_web`.

A course has chapters like a book, where as a website version, will appear more like a typical website.
```

Reply:
hmmm I feel like we might need additional guidance about how these differ